src/Entity/Styles/chicago-fullnote-bibliography.xml

```diff
@@ -0,0 +1,1459 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What did you change in this file from the original?

Reply:
Alot

Reply:
Document the lot.